packages/components/tabs/style/themes/default.variable.less

```
@@ -3,12 +3,12 @@
 @tabs-active-color: @color-primary-d10;
 @tabs-disabled-color: @color-graphite-l10;
 
-@tabs-nav-font-size: @font-size-sm;
+@tabs-nav-font-size-md: var(--ix-font-size-sm);
+@tabs-nav-font-size-lg: var(--ix-font-size-md);
```

```suggestion
@tabs-nav-font-size-md: var(--ix-font-size-md);
@tabs-nav-font-size-lg: var(--ix-font-size-lg);
```

default 的字体默认应该比 seer 的大一号。